tensorbay/cli/auth.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Implementation of gas auth."""
+
+from typing import Dict
+
+from .utility import error, form_profile_value, read_config, update_config, write_config
+
+
+def _implement_auth(obj: Dict[str, str], arg1: str, arg2: str) -> None:
+    update_config()
+    config_parser = read_config()
+
+    if (not arg1 and not arg2) or (_is_url(arg1) and not arg2):
+        error("Require accessKey to authenticate Tensorbay account")
+
+    if _is_accesskey(arg1) and _is_url(arg2):
+        error('Please use "gas config [url] [accessKey]" to specify the url and accessKey')
+
+    if _is_url(arg1) and _is_accesskey(arg2) or _is_accesskey(arg1) and not arg2:
```

Comment:
what if the input is `gas auth <accessKey> <somethingIllegal>` like:
`gas auth Accesskey-xxxxxxx asfhaphgd`

Reply:
It falls into the `else` case.